src/lsp/cobol_parser/grammar_tokens.mly

```diff
@@ -138,8 +138,8 @@
 %token BITMAP_TRAILING              [@keyword]                [@contexts ]
 %token BITMAP_TRANSPARENT_COLOR     [@keyword]                [@contexts ]
 %token BITMAP_WIDTH                 [@keyword]                [@contexts ]
-%token BLANK                        [@keyword]
-%token BLINK                        [@keyword]                [@contexts screen_descr_entry, set_attribute_stmt]
+%token BLANK                        [@keyword]                [@contexts display_stmt, accept_stmt]
```

Comment:
I think that one should not become context-sensitive.

Reply:
I removed the context on `BLANK`.